### PR TITLE
Tests for minor children card and group

### PR DESCRIPTION
**Pull request by lavanyareddyreddy**
#### What tests were added (if relevant), and do all tests (rspec, karma, and integration CALS tests) pass?
-yes, minorCardGroup_tests.js and minorChildrenCard_tests.js

#### Which User story does this relate to?
-#147206857

#### Brief feature description
- updated jasmine tests for minor card field and minor card group.
remove firstname, middle name and last name from minor card.

#### Does this PR block/effect other stories?
- no

#### Do any commands needs to be run in order to run the project with this PR?
- npm run karma

#### Code coverage percent diff as a result of this PR?
- js: 64.09%

#### JS errors diff as a result of this PR?
- n/a

### CODE REVIEW

**Comment by ratneshraval** [comment]
Looks like this isn't branched from development. Can you merge development in here.

File: test/javascript/rfa_forms/rfa01a_edit_view/minorCardGroup_tests.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+      spyOn(component.instance(), 'addCard').and.callThrough()
+      component.instance().addCard()
+      expect(component.instance().addCard).toHaveBeenCalled()
+
+      // build data that parent should be called with
+      let newData = []
+      newData[0] = minorCardChild
+      newData[1] = minorDefaults
+
+      expect(props.minorchil.length).toEqual(1)
```
Comment (ratneshraval) [inline test/javascript/rfa_forms/rfa01a_edit_view/minorCardGroup_tests.js]: The length should be 2 or validate using .toHaveBeenCalledWith.

File: test/javascript/rfa_forms/rfa01a_edit_view/minorCardGroup_tests.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+    describe('when close minor card is clicked', () => {
+      it('Clears data when 1 phone number is present', () => {
+        spyOn(component.instance(), 'clickClose').and.callThrough()
+        component.find('.glyphicon-remove').simulate('click')
+        let newData = []
+        newData[0] = minorCardChild
+        newData[1] = minorDefaults
+        expect(component.instance().clickClose).toHaveBeenCalledWith(0)
+
```
Comment (ratneshraval) [inline test/javascript/rfa_forms/rfa01a_edit_view/minorCardGroup_tests.js]: Please validate that parent is being called with blank data.

Comment (lavanyareddyreddy) [inline test/javascript/rfa_forms/rfa01a_edit_view/minorCardGroup_tests.js]: sure validating with minorDefaults now.

**Comment by ratneshraval** [comment]
@lavanyareddyreddy it is sending blank DOB when I didn't type anything.

**Comment by lavanyareddyreddy** [comment]
@ratneshraval , what do you expect when you don't enter anything in dob field. are we supposed to send nil? dob is not dictionary it is empty when you don't enter any value.

**Comment by ratneshraval** [comment]
It shouldn't send anything. You have default dob as blank, so it is sending blank. If you remove it, it should fix it.

**Pull request merged**